catalog/proguard-rules.pro

```diff
@@ -0,0 +1,11 @@
+-keepattributes *Annotation*, InnerClasses
+-dontnote kotlinx.serialization.AnnotationsKt # core serialization annotations
+
```

Comment:
why do we need these rules?

Reply:
They are generated. At this point, I want to change as little as possible in the generated files.